sources/dynamodb/schema_test.go

```diff
@@ -459,12 +608,42 @@ func TestListTables(t *testing.T) {
 	client := &mockDynamoClient{
 		listTableOutputs: listTableOutputs,
 	}
-
-	tables, err := listTables(client)
+	isi := InfoSchemaImpl{client, 10}
+	tables, err := isi.GetTables()
 	assert.Nil(t, err)
-	assert.Equal(t, []string{"table-a", "table-b"}, tables)
+	assert.Equal(t, []common.SchemaAndName{{"", "table-a"}, {"", "table-b"}}, tables)
 }
 
+func TestInfoSchemaImpl_GetTableName(t *testing.T) {
+	tableNameA := "table-a"
+
+	client := &mockDynamoClient{}
+	isi := InfoSchemaImpl{client, 10}
+	table := isi.GetTableName("", tableNameA)
+	assert.Equal(t, tableNameA, table)
+}
+
+//
```

Comment:
remove commented code

Reply:
This is a WIP. I'll populate these tests in this PR shortly.